docs/sql-migration-guide-upgrade.md

```diff
@@ -50,6 +50,8 @@ license: |
 
   - In Spark version 2.4 and earlier, JSON datasource and JSON functions like `from_json` convert a bad JSON record to a row with all `null`s in the PERMISSIVE mode when specified schema is `StructType`. Since Spark 3.0, the returned row can contain non-`null` fields if some of JSON column values were parsed and converted to desired types successfully.
 
+  - Refreshing a cached table would trigger a table uncache operation and then a table cache (lazily) operation. In Spark version 2.4 and earlier, the cache name ane cache storage level are not preserved before the uncache operation. Therefore, the cache name and cache storage level could be changed unexpectedly. Since Spark 3.0, cache name and cache storage level will be first preserved for cache recreation. It helps to maintain a consistent cache behavior upon table refreshing.
```

Comment:
`ane` -> `and`

Reply:
Hi @gatorsmile , thanks. just fixed the typo.